### PR TITLE
[hotfix][runtime][tests] Modify spelling error in RestServerSSLAuthITCase.java

### PR DESCRIPTION
**Pull request by LongWangXX**
## What is the purpose of the change

Fix clinet typo in RestServerSSLAuthITCase class in tests section in `flink-runtime` module

## Brief change log

Replace clinet in the constructor of the RestServerSSLAuthITCase class with client

## Verifying this change

Please make sure both new and modified tests in this PR follows the conventions defined in our code quality guide: https://flink.apache.org/contributing/code-style-and-quality-common.html#testing

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): ( no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no know)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
@LongWangXX Thanks for the PR. Please have a look at https://flink.apache.org/contributing/contribute-code.html for the guide on how to contribute, specifically the commit naming convention and update your commit message to match this.

**Comment by LongWangXX** [comment]
@MartijnVisser okay, thank you!

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
@LongWangXX Thanks for your PR. The change itself in the code looks good, but the commit message isn't yet and therefore I can't merge it yet. 

You currently have `[typo clinet]` as commit message, but this should be something like `[hotfix][runtime][tests] Modify spelling error in RestServerSSLAuthITCase.java`

If you need any help with changing your commit message, do let me know.

**Comment by LongWangXX** [comment]
@MartijnVisser Okay, thanks for the notification. I'll fix it right away.

**Comment by MartijnVisser** [comment]
@LongWangXX I don't see the changed commit message in this PR https://github.com/apache/flink/pull/19265/commits

**Comment by LongWangXX** [comment]
@MartijnVisser Sorry, I misunderstood just now. Do you see if this is possible?

**Comment by MartijnVisser** [comment]
@LongWangXX This is not exactly what I meant, because now we have 3 commits. It should have been one with the changed commit message. It now requires a rebase. I'll help you out in a second.

**Comment by MartijnVisser** [comment]
@LongWangXX Unfortunately I can't push to your branch. What you need to do is run the following command:

1. Make sure that you have Flink's Git repo added as a Git remote. You can verify that by running `git remote -v`. It should display a link to your fork (most likely called `origin`). It should also have a link to apache/flink. If that's not there, you can add it by running `git remote add upstream git@github.com:apache/flink.git`. If you now run `git remote -v` you will see two Git remotes. 
1. Checkout your local branch
2. Run `git fetch upstream` to fetch all changes
3. Run `git rebase upstream/master` to rebase your local branch to Flink's master branch
4. Run a `git push --force` to push your rebased local branch

**Comment by MartijnVisser** [comment]
OK now there's one more thing to fix, the commit message itself. You can do that by:

1. Run `git reset --soft HEAD~1` to reset your Git index to the last commit
2. Run `git add --all` to add all the files
3. Run `git commit` to commit the same change, this time use the correct Git commit message
4. Run `git push --force` to push the changed commit message in

**Comment by LongWangXX** [comment]
@MartijnVisser Thank you very much for your patient guidance.

**Review by MartijnVisser — APPROVED** [review]
@LongWangXX Nice! LGTM once the CI passes (which shouldn't be an issue, but I'll check in a couple of hours)

**Comment by LongWangXX** [comment]
@MartijnVisser Ok, thank you~ Can this be merged? ☺

**Comment by MartijnVisser** [comment]
@flinkbot run azure

**Comment by MartijnVisser** [comment]
@flinkbot run azure

**Comment by MartijnVisser** [comment]
CI passed in https://dev.azure.com/apache-flink/98463496-1af2-4620-8eab-a2ecc1a2e6fe/_build/results?buildId=33958# - Merging this PR

**Pull request merged**
